### ISSUE TITLE
Score fusion of iDT with SeqVLAD？

### ISSUE THREAD

**Issue by ZealACMer** [youjiangxu/seqvlad-pytorch#3]
How did you fuse the iDT based scores with the scores of your method, thank you so much. Looking forward to your answer.

**Comment by youjiangxu** [comment]
Hi，
      The details about fusing the iDT based scores are in the file "merge_hmdb.py". I think you should add the path to the idt score files "--idt" and the source decision values can be obtained from Gül Varol Simsekli <gul.varol@inria.fr>. And I also upload a backup in the 'idt_decision_values' file.  

Please use python2 to run 'merge_hmdb.py --rgb <rgb logits path> --flow <flow logits path> --idt <idt logits path, a mat format file> --split <1, 2 or 3>'

Thank you!

**Comment by ZealACMer** [comment]
Thank you so much for your prompt reply. I think your fusion strategy about iDT and SeqVLAD is late fusion, which is performed by calculating the weighted average of classification scores of IDT with the scores predicted by your proposed method SeqVLAD for each class of videos. Then the label with the highest value is utilized as the predicted label. Am I right?

**Comment by youjiangxu** [comment]
Yes, you are right.

**Comment by ZealACMer** [comment]
Thanks a lot.
